src/ranges.jl

```diff
@@ -44,6 +44,8 @@ known_step(::Type{<:AbstractUnitRange{T}}) where {T} = one(T)
 
 _get(x) = x
 _get(::Val{V}) where {V} = V
```

Comment:
Do we still need to support `Val` now that we're switching over to `Static` and `Val` can't be any of the fields in `OptionallyStaticUnitRange` anymore?

Reply:
I've removed it.